.github/workflows/ci.yml

```
@@ -0,0 +1,42 @@
+
+on:
+  push:
+  pull_request:
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    services:
+    steps:
+    - uses: actions/checkout@v4
+    - name: Set up Ruby
```

actionlint (1.7.9)

step must run script with "run" section or run action with "uses" section

(syntax-check)